Dockerfile

```diff
@@ -17,10 +17,11 @@ RUN \
     curl -sL https://deb.nodesource.com/setup_10.x | bash -; \
     apt-get install -y nodejs; \
     cd kinto/plugins/admin; npm install; npm run build; \
+    pip3 install --upgrade pip; pip3 install httpie; \
     pip3 install -e /app[postgresql,memcached,monitoring] -c /app/requirements.txt; \
     pip3 install kinto-attachment kinto-emailer kinto-elasticsearch kinto-portier kinto-redis; \
     kinto init --ini $KINTO_INI --host 0.0.0.0 --backend=memory --cache-backend=memory; \
-    apt-get purge -y -qq gcc libssl-dev libffi-dev libpq-dev curl nodejs; \
+    apt-get purge -y -qq gcc libssl-dev libffi-dev libpq-dev curl nodejs httpie; \
```

Comment:
You didn't install httpie from apt, so this won't work.

Reply:
I respect the simplicity so that to avoid complexity I revert all changes to the first place and the only changes still remain in `Dockerfile`, contain two segments: `httpie` right after `nodejs` at line 24 and the other upgrading `pip` by 

> `pip3 install pip --upgrade`

 before installing packages by `pip3`. It's necessary to upgrade `pip3` in which it reports a warning during the installation process.
All suggestions are postponing to next pull request that I'm gonna update the document to aim Microsoft Windows users and add a `Powershell` script to run the application under the hood.

Reply:
Thanks

Reply:
Can you elaborate the change on httpie?

Reply:
> You didn't install httpie from apt, so this won't work.

It doesn't matter because of the `docker` environment. You're monitoring part of the `HTTP` traffic from where it's not necessarily internal and doesn't depend on the version of `Python`. The system installs the `httpie` with respect to `Python2` environment and `pip3` does it correspond to `Python3` core. If you're gonna import `httpie` as a module and write some tests, obviously, `pip3` is middling otherwise the only thing that command line cares about is consistency, bottom line.

Reply:
Furthermore, the main idea to append the `httpie` into Dockerfile is the nature of *Kinto* to communicate over http(s) protocol. So, it seems attaching `httpie` to the container helps developers to be more equipped in order to test the application omnidirectionally.
Besides, it actually doesn't matter to be installed by either of `pip3` and `apt-get` because it's a handy tool to make get/post/put/delete/patch requests more easily. As I mentioned, if the purpose of adding `httpie` has to cover all aspects, that is, test stuff throughout the command line and pure python scripts, yeah, both `apt-get` and `pip3` should have been involved unless it would be enough to choose one amongst

> pip3 install pip --upgrade && apt-get httpie

where the latter preferably will be installed based on `Python2` core if it's already installed. If `pip3` after `apt-get` installs the `httpie` package and `httpie` has been installed and attached to `PATH` it does not just override the executable file but also install the eggs and it's ready to be imported in python program. Also, if you choose to install it by means of `pip3` it should be done in the following fashion why the upgrade has to be completed before the `pip3` starts to install `httpie`

> pip3 install pip --upgrade
> pip3 install httpie

Any `Python` interpreter has its own specific version of `pip` whatever the major and minor version of all `Python` interpreters are the same so that `pip` needs to be upgraded to the last stable version otherwise causes reporting to warn you to upgrade the `pip`; this was the why I offered you to append both segments including `pip3 install --upgrade pip` and `pip3 install httpie`, however, according to discussion `httpie` installation has no unique way.
I hope that is persuasive and we can finally move on and you merge this PR and go forward to the next PR.

Reply:
> You didn't install httpie from apt, so this won't work.

Yeah, you right. It was awkwardly had been missed. BTW, it's done.